WBooks/TableViewCell.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  TableViewCell.swift
+//  WBooks
+//
+//  Created by Nicolas Manograsso on 08/10/2018.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import UIKit
+
+class TableViewCell: UITableViewCell {
+
+    
+    @IBOutlet weak var lblTitle: UILabel!
+    @IBOutlet weak var lblAuthor: UILabel!
+    @IBOutlet weak var imgBook: UIImageView!
+    
+    
+    func setText(book: Book){
+        lblTitle.text = book.title
+        lblAuthor.text = book.author
+        let imageURL : String?
```

Comment:
you can do this in one line: `let imageUrl = book.image`, you don't need to specify the type.